ctf/check.cmd

```diff
@@ -0,0 +1,12 @@
+@echo off
```

Comment:
Batch script is used to run all suites.
So far I developed a cmd to run each suite individually. There is only one suite now, but I expect approximately between 50 and 100 suites. I'd like to put them all in batch to check everything with simple "check" command.

So, what is wrong with batch script?
If you don't like it, don't use it. You are still able to open the terminal and run whatever you want to run.
You are free to add .sh one for your system.
Why do you want to complicate my life?